Makefile.PL

```diff
@@ -71,3 +79,10 @@ WriteMakefile(
         },
     },
 );
+
+if (!eval { ExtUtils::MakeMaker->VERSION('6.6303') }) {
```

Comment:
Sorry, I was reading this on my mobile and couldn't see the commit message,
didn't see the whole EUMM bit either. GitHub mobile is not without
limitation it seems :-)

Didn't think about contributor names, good catch. Also, I got the EUMM bit
backwards in my brain. Thanks for answering.

Looks like Travis likes your changes as well, thanks for this!
On Sun, Oct 18, 2015 at 17:30 Karen Etheridge <notifications@github.com>
wrote:

> In Makefile.PL
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/85#discussion_r42326721>
> :
>
> > @@ -71,3 +79,10 @@ WriteMakefile(
> >          },
> >      },
> >  );
> > +
> > +if (!eval { ExtUtils::MakeMaker->VERSION('6.6303') }) {
>
> I don't understand what you mean. This line is not requiring that version
> of EUMM -- it's providing code that will work if this version *isn't*
> installed on the user side. TEST_REQUIRES was not added until EUMM
> 6.63_03; on versions before that, BUILD_REQUIRES is the best we had.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/85/files#r42326721>
> .
>